apollo-portal/src/main/resources/static/app/access_key.html

```diff
@@ -0,0 +1,168 @@
+                                </button>
+                            </form>
+                        </div>
+                    </section>
+
+                    <!--application info-->
+                    <section  ng-repeat="env in envs" ng-value="env">
+                        <hr>
+                        <h4>{{'Common.Environment' | translate }}: {{env}}
+                        </h4>
+
+                        <table class="table table-striped table-hover table-bordered">
+                            <thead>
+                                <tr>
+                                    <th>{{'AccessKey.ConfigAccessKeys.Secret' | translate }}</th>
+                                    <th>{{'AccessKey.ConfigAccessKeys.Status' | translate }}</th>
+                                    <th>{{'AccessKey.ConfigAccessKeys.LastModify' | translate }}</th>
+                                    <th>{{'AccessKey.ConfigAccessKeys.LastModifyTime' | translate }}</th>
+                                    <th>{{'AccessKey.ConfigAccessKeys.Operator' | translate }}</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr ng-show="(!accessKeys[env] || accessKeys[env].length < 1)">
+                                    <td colspan="5" style="text-align: center;">{{'AccessKey.NoAccessKeyServiceTips' | translate }}</td>
+                                </tr>
+                                <tr ng-show="accessKeys[env] && accessKeys[env].length > 0"
+                                    ng-repeat="accessKey in accessKeys[env]">
+                                    <td style="text-align: center;">{{accessKey.secret}}</td>
+                                    <td style="text-align: center;" ng-style="{'color': accessKey.enabled ? '#5cb85c' : '#d20707'}">{{accessKey.enabled ? ('AccessKey.Operator.Enabled' | translate) : ('AccessKey.Operator.Disabled' | translate) }}</td>
+                                    <td style="text-align: center;">{{accessKey.dataChangeLastModifiedBy}}</td>
+                                    <td style="text-align: center;">{{accessKey.dataChangeLastModifiedTime | date: 'yyyy-MM-dd HH:mm:ss'}}</td>
+                                    <td style="text-align: center;">
+                                        <a href="javascript:;"
+                                           ng-click="enable(accessKey.id, env)" ng-if="!accessKey.enabled">{{'AccessKey.Operator.Enable' | translate}}</a>
+                                        <a href="javascript:;"
+                                           ng-click="disable(accessKey.id, env)" ng-if="accessKey.enabled">{{'AccessKey.Operator.Disable' | translate}}</a>
+                                        <a href="javascript:;"
+                                           ng-click="remove(accessKey.id, env)">{{'AccessKey.Operator.Remove' | translate }}</a>
```

Comment:
since the remove will take up to 1 minute to take effect, shall we only allow removing disabled secret keys? e.g. when the user wants to remove an enabled key, we promt him to disable the sceret first?

Reply:
That's a good idea 👍!!!

Reply:
How about hiding the `Remove` button directly when it is enabled?

Reply:
![image](https://user-images.githubusercontent.com/8198862/71571556-d66d0d00-2b15-11ea-9832-4c5dc45f5fa1.png)

Reply:
This is a good idea.